### PR TITLE
Create file price (xml,json)

### PR DESCRIPTION
**Pull request by prohoroff**
Test pull request)

### CODE REVIEW

File: price-calc/src/main/java/by/prohorov/price/kg/CostByPriceKgUseProperties.java [hunk 1]
```diff
@@ -33,8 +33,8 @@ public void setCostWeight(double costWeight) {
     public double costByPriceWeightInKg() {
         Properties price = new Properties();
         try {
-            ClassLoader classLoader = CostByPriceKgUseFile.class.getClassLoader();
-            InputStream is = classLoader.getResourceAsStream("price_kg.properties");
+            ClassLoader classLoader = CostByPriceKgUseProperties.class.getClassLoader();
+            InputStream is = classLoader.getResourceAsStream("price/price_kg.properties");
```
Comment (zhurlik) [inline price-calc/src/main/java/by/prohorov/price/kg/CostByPriceKgUseProperties.java]: https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html

File: price-calc/src/main/java/by/prohorov/price/entity/Price.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+import java.util.ArrayList;
+
+/**
+ * Created by Artsiom Prokharau 09.02.2021
+ */
+
+public class Price {
+
+    private Integer light;
```
Comment (zhurlik) [inline price-calc/src/main/java/by/prohorov/price/entity/Price.java]: what happens when `Integer` will be nulls

File: price-calc/src/main/java/by/prohorov/price/kg/CostByPriceKgUseFileXml.java [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+    public void setCostWeight(double costWeight) {
+        this.costWeight = costWeight;
+    }
+
+    @Override
+    public double costByPriceWeightInKg() {
+        try {
+            if (costWeight < 4) {
+                costWeight = Jackson.loaderXmlFile().getLight();
```
Comment (zhurlik) [inline price-calc/src/main/java/by/prohorov/price/kg/CostByPriceKgUseFileXml.java]: how will you handle the case when `Jackson` will not be able to load the needed data

File: price-calc/src/main/java/by/prohorov/price/kg/CostByPriceKgUseProperties.java (hunk 1 above)
Comment (prohoroff) [reply price-calc/src/main/java/by/prohorov/price/kg/CostByPriceKgUseProperties.java]: Hi. I'm corrected. Now I often use it.

File: price-calc/src/main/java/by/prohorov/price/entity/Price.java (hunk 2 above)
Comment (prohoroff) [reply price-calc/src/main/java/by/prohorov/price/entity/Price.java]: Created validatePrice

File: price-calc/src/main/java/by/prohorov/price/kg/CostByPriceKgUseFileXml.java (hunk 3 above)
Comment (prohoroff) [reply price-calc/src/main/java/by/prohorov/price/kg/CostByPriceKgUseFileXml.java]: Add check Exception to class Jackson

Comment (zhurlik) [reply price-calc/src/main/java/by/prohorov/price/kg/CostByPriceKgUseFileXml.java]: I don't like a lot of try-catch/exception, why you don't want for example set integers = 0 (by default), then you can return an empty `new Price()`. I mean you will not check nulls and will avoid unnecessary try-catch

**Pull request merged**